python/artm/master_component.py

```diff
@@ -23,6 +23,8 @@ def _regularizer_type(config):
         return constants.RegularizerType_SmoothPtdw
     elif isinstance(config, messages.TopicSelectionThetaConfig):
         return constants.RegularizerType_TopicSelectionTheta
+    elif isinstance(config, messages.iTopicThetaConfig):
+        return constants.RegularizerType_iTopicTheta
 
```

Comment:
@bt2901 Sorry, I missed this question. Sure, `` raise NotImplementedError`` is a good idea. I'm not sure how this normally expressed in python -- so choose any pythonic way, up to you :)